services/ui-src/src/components/reports/StandardReportPage.tsx

```diff
@@ -9,51 +9,28 @@ import {
   ReportPageIntro,
 } from "components";
 // utils
-import { filterFormData, useFindRoute, useUser } from "utils";
-import { AnyObject, StandardReportPageShape, ReportStatus } from "types";
+import { useFindRoute } from "utils";
+import { StandardReportPageShape } from "types";
 
 export const StandardReportPage = ({ route }: Props) => {
-  const [submitting, setSubmitting] = useState<boolean>(false);
-  const { report, updateReport } = useContext(ReportContext);
-  const { full_name, state, userIsStateUser, userIsStateRep } =
-    useUser().user ?? {};
+  const { report } = useContext(ReportContext);
   const navigate = useNavigate();
   const { nextRoute } = useFindRoute(
     report!.formTemplate.flatRoutes!,
     report!.formTemplate.basePath
   );
 
-  const onSubmit = async (enteredData: AnyObject) => {
-    if (userIsStateUser || userIsStateRep) {
-      setSubmitting(true);
-      const reportKeys = {
-        state: state,
-        id: report?.id,
-      };
-      const filteredFormData = filterFormData(enteredData, route.form.fields);
-      const dataToWrite = {
-        metadata: {
-          status: ReportStatus.IN_PROGRESS,
-          lastAlteredBy: full_name,
-        },
```

Comment:
Blocking: The new autosave update methods in each field have a problem where they aren't saving the metadata because they aren't wrapped in a `metadata: {}` object like this, so `status` and `lastAlteredBy` are getting ignored by the API handler and never sent to the database.

Reply:
Update: I fixed all these in my commit here: https://github.com/CMSgov/mdct-mcr/pull/5336/commits/7bb0cbc4e95781c3724b99d5157f54c66166121a